projects/wvr-elements/src/lib/wvr-nav-list/wvr-nav-li/wvr-nav-li.component.ts

```diff
@@ -1,19 +1,24 @@
-import { Component, Input } from '@angular/core';
-import { wvrThemeHostBindings, wvrThemeInputs } from '../../shared/wvr-styles';
-
+import { Component, HostBinding, Input } from '@angular/core';
+/**
+ * The WvrNavLi Component contains the href property of the list.
```

Comment:
The WvrNavLi Component is the list element to be used with the wvr-nav-list element.

Reply:
Applied correction for this comment and also for the remainder properties as well